src/main/java/eu/decentsoftware/holograms/event/DecentHologramsRegisterEvent.java

```diff
@@ -0,0 +1,32 @@
+package eu.decentsoftware.holograms.event;
+
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+
+/**
+ * This event is called after the DecentHolograms plugin is reloaded.
+ *
+ * @author d0by
+ * @since 2.7.8
+ */
```

Comment:
Please don't just copy-paste comments... Actually update them too, as this one here is misleading in multiple ways as it

- Tells completely wrong information
- Has the wrong `@author` set (d0by didn't make this class) and
- Has the wrong `@since` set (Current version is 2.8.6, not 2.7.8)